src/main/java/com/puppycrawl/tools/checkstyle/checks/coding/FallThroughCheck.java

```diff
@@ -70,74 +70,70 @@
  * Example:
  * </p>
  * <pre>
- * public void foo() throws Exception {
- *   int i = 0;
- *   while (i &gt;= 0) {
- *     switch (i) {
- *       case 1:
- *         i++;
- *       case 2: // violation, previous case contains code but lacks
- *               // break, return, yield, throw or continue statement
- *         i++;
- *         break;
- *       case 3: // OK
- *         i++;
- *         return;
- *       case 4: // OK
- *         i++;
- *         throw new Exception();
- *       case 5: // OK
- *         i++;
- *         continue;
- *       case 6: // OK
- *       case 7: // Previous case: OK, case does not contain code
- *               // This case: OK, by default the last case might not have statement
- *               // that transfer control
- *         i++;
+ * class FallThroughTestClass {
+ *   public static void main(String args[]) {
+ *     int i = 0;
+ *     while(i &gt;= 0) {
+ *       switch(i) {
+ *         case 1:
+ *           i++;
+ *         case 2: // violation, previous case contains code but lacks
+ *           // break, return, yield, throw or continue statement
+ *           i++;
+ *           break;
+ *         case 3: // OK
+ *           i++;
+ *           return;
+ *         case 4: // OK
+ *           i++;
+ *           throw new Exception();
+ *         case 5: // OK
+ *           i++;
+ *           continue;
+ *         case 6: // OK
+ *         case 7: // fall through
+ *         case 8: // Previous case: OK, case does not contain code
+ *           // This case: OK, by default the last case might not have statement
+ *           // that transfer control
```

Comment:
Please remove all trailing comments

Reply:
So do want it to be like this?
`case 8: // Previous case: OK, case does not contain code. This case: OK, by default the last case might not have statement that transfer control`